crates/task-impls/src/events.rs

```diff
@@ -46,15 +46,15 @@ pub enum HotShotEvent<TYPES: NodeType> {
     TimeoutVoteRecv(TimeoutVote<TYPES>),
     /// Send a timeout vote to the network; emitted by consensus task replicas
     TimeoutVoteSend(TimeoutVote<TYPES>),
-    /// A DA proposal has been received from the network; handled by the DA task
+    /// A Da proposal has been received from the network; handled by the Da task
     DaProposalRecv(Proposal<TYPES, DaProposal<TYPES>>, TYPES::SignatureKey),
-    /// A DA proposal has been validated; handled by the DA task and VID task
+    /// A Da proposal has been validated; handled by the Da task and VID task
     DaProposalValidated(Proposal<TYPES, DaProposal<TYPES>>, TYPES::SignatureKey),
-    /// A DA vote has been received by the network; handled by the DA task
+    /// A Da vote has been received by the network; handled by the Da task
     DaVoteRecv(DaVote<TYPES>),
-    /// A Data Availability Certificate (DAC) has been recieved by the network; handled by the consensus task
+    /// A Data Availability Certificate (DaC) has been recieved by the network; handled by the consensus task
```

Comment:
completely inconsequential nit: I think this should either be `Dac` or `DAC` or `DaCertificate` but not `DaC` (by analogy with `UUID` -> `Uuid`)

Reply:
Personally I think acronyms in comments should be the same as in written text, so all capitalized.

Reply:
hah! Stupid find/replace... fixed!

Reply:
Sorry @bfish713 I'm not sure how this comment didn't get loaded, I only saw Salman's. I can swap the comments back

Reply:
I really have no strong preference between any of the other options, except that I thought `DaC` specifically looked weird. I think `DAC` is what I would write too